### PR TITLE
new module svtk/rdtest2vcf

### LINKED ISSUE #2376: new module: svtk/rdtest2vcf
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #2376 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/svtk/rdtest2vcf/main.nf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    tuple val(meta), path("*.vcf.gz")       , emit: vcf
+    tuple val(meta), path("*.vcf.gz.tbi")   , emit: tbi
+    path "versions.yml"                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
```
Comment (jfy133) [inline modules/nf-core/svtk/rdtest2vcf/main.nf:24]:
```suggestion
```

File: modules/nf-core/svtk/rdtest2vcf/main.nf [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    def VERSION = '0.0.20190615' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+
```
Comment (jfy133) [inline modules/nf-core/svtk/rdtest2vcf/main.nf:26]:
```suggestion
```

File: modules/nf-core/svtk/rdtest2vcf/meta.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+name: "svtk_rdtest2vcf"
+description: Convert an RdTest-formatted bed to the standard VCF format.
+keywords:
+  - sort
```
Comment (jfy133) [inline modules/nf-core/svtk/rdtest2vcf/meta.yml:4]: more keywords please!

File: modules/nf-core/svtk/rdtest2vcf/meta.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+tools:
+  - "svtk":
+      description: "Utilities for consolidating, filtering, resolving, and annotating structural variants."
+      homepage: "https://github.com/broadinstitute/gatk-sv/tree/master/src/svtk"
+      documentation: "https://github.com/broadinstitute/gatk-sv/tree/master/src/svtk"
+      tool_dev_url: "https://github.com/broadinstitute/gatk-sv/tree/master/src/svtk"
+      doi: ""
```
Comment (jfy133) [inline modules/nf-core/svtk/rdtest2vcf/meta.yml:11]: For DOi see bottom of this page: https://gatk.broadinstitute.org/hc/en-us/articles/9022653744283-GATK-Best-Practices-for-Structural-Variation-Discovery-on-Single-Samples

File: modules/nf-core/svtk/rdtest2vcf/meta.yml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bed:
+      type: file
+      description: The RdTest-formatted bed
```
Comment (jfy133) [inline modules/nf-core/svtk/rdtest2vcf/meta.yml:22]:
```suggestion
      description: A RdTest-formatted bed
```

File: modules/nf-core/svtk/rdtest2vcf/meta.yml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+      pattern: "*.bed"
+  - samples:
+      type: file
+      description: A text file containing the names of all samples that need to be added to the VCF
+      pattern: "*.txt"
+
```
Comment (jfy133) [inline modules/nf-core/svtk/rdtest2vcf/meta.yml:28]: Missing fasta_fai input

**Review by jfy133 — APPROVED** [review]
Just docs updates, so once they are in you can merge :+1:

File: modules/nf-core/svtk/rdtest2vcf/meta.yml (hunk 3 above)
Comment (nvnieuwk) [reply modules/nf-core/svtk/rdtest2vcf/meta.yml:4]: I always manage to forget those :grin:

**Pull request merged**